src/domain/alerts/alerts.repository.ts

```diff
@@ -26,8 +26,8 @@ export class AlertsRepository implements IAlertsRepository {
     private readonly alertsApi: IAlertsApi,
     @Inject(IEmailApi)
     private readonly emailApi: IEmailApi,
-    @Inject(IEmailRepository)
-    private readonly emailRepository: IEmailRepository,
+    @Inject(IAccountRepository)
+    private readonly emailRepository: IAccountRepository,
```

Comment:
I'd rename this variable also to `accountRepository`

Reply:
[fa63a4e](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/fa63a4e9950036c1b044c936774359b805c7c62f)